std/typecons.d

```diff
@@ -5509,19 +5509,19 @@ if (!is(T == class) && !(is(T == interface)))
             import core.exception : onOutOfMemoryError;
             import core.stdc.string : memcpy, memset;
 
-            _store = cast(Impl*) pureMalloc(Impl.sizeof);
+            _store = (() @trusted => cast(Impl*) pureMalloc(Impl.sizeof))();
             if (_store is null)
                 onOutOfMemoryError();
             static if (hasIndirections!T)
-                pureGcAddRange(&_store._payload, T.sizeof);
+                (() @trusted => pureGcAddRange(&_store._payload, T.sizeof))();
```

Comment:
`@trusted` for the pureGcAddRange. Okay because this cannot cannot conceivably cause memory corruption (pointer is non-null and size is correct).